addons/l10n_es_edi_facturae/models/account_move.py

```diff
@@ -234,7 +234,7 @@ def _l10n_es_edi_facturae_inv_lines_to_items(self, conversion_rate=None):
             price_before_discount = sum(to_update['price_subtotal'] for _dummy, to_update in tax_before_discount['base_lines_to_update'])
             discount = max(0., (price_before_discount - line.price_subtotal))
             surcharge = abs(min(0., (price_before_discount - line.price_subtotal)))
-            totals['total_gross_amount'] += price_before_discount
+            totals['total_gross_amount'] += line.price_subtotal
             totals['total_general_discounts'] += discount
             totals['total_general_surcharges'] += surcharge
```

Comment:
```suggestion
```

Reply:
Me and CHKL thought it's a good idea to only fix the issues the clients are currently talking about right now, and fix the general discounts/charges in another commit as they also need to be computed correctly. But for now we might as well remove those two lines. We also discussed more yesterday and there will be more fixes in this commit.

Reply:
@chklop @alialfie Ok, it is linked.